marshmallow_recipe/generics.py

```diff
@@ -38,8 +38,9 @@ def get_fields_type_map(cls: type) -> FieldsTypeMap:
 
     class_type_var_map = get_class_type_var_map(cls)
     fields_class_map = get_fields_class_map(origin)
+    field_type_hints = typing.get_type_hints(origin, include_extras=True)
```

Comment:
`typing.get_type_hints` resolves string-defined typings. If you believe that it can cause a performance hit we can easily call this only in cases when we have such string typings

Reply:
Nope, this is alright. We do that only once, so not an issue.